TwitchDownloaderWPF/WindowUrlList.xaml

```diff
@@ -4,16 +4,30 @@
         xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
```

Comment:
Could you convert this file from tabs to 4 spaces? Most of the files are 4 spaces but it seems some are still tabs.

Reply:
That's my preference as well, just didn't want to change the status quo; will remove the conversion back to tabs.